README.md

```diff
@@ -0,0 +1,58 @@
+# Awesome books: plain JavaScript with classes
+
+- In this project, we build a basic website that allows users to add/remove books from a list. 
+- We achieve that by using JavaScript classes and arrays. 
+- We also dynamically modify the DOM and added basic events.
+
+
+
+## Built With
+- Major languages (HTML, CSS , JavaScript)
+- Frameworks (Bootstrap)
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+**This is the Awesome books: plain JavaScript with classes**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+    Web Browser (e.g. Firefox, Chrome, Edge)
+
+### Setup
+    -using git command line
+      if you are using the git command line, please use this command in your terminal
+        git clone git@github.com:Kelvin-Ben/Awesome-Books.git
+    -using the GitHub Desktop
+      if you are using the desktop version of github, please  click on the green "code" button and click on "Open with GitHub Desktop" 
+
+
+### Install
+    Install the linters in your local environment to mantain your code clean 
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] If you don't have something to say in these section, it would be better if you remove them to make your README  look clean 😎